test/sanity/code-smell/package-data.py

```diff
@@ -36,13 +37,13 @@ def collect_sdist_files(complete_file_list: list[str]) -> list[str]:
         'SOURCES.txt',
         'dependency_links.txt',
         'entry_points.txt',
-        'not-zip-safe',
         'requires.txt',
         'top_level.txt',
     )
 
     sdist_files.append('PKG-INFO')
-    sdist_files.extend(f'lib/ansible_core.egg-info/{name}' for name in egg_info)
+    sdist_files.append('setup.cfg')
```

Comment:
Why is this added? Should it be dropped?
```suggestion
```

Reply:
It's generated during the build and included in the sdist, which is why it's added to the list of files to expect.

Reply:
Oh, is that for some ancient envs?

Reply:
It appears to be the default behavior in setuptools. See https://github.com/pypa/setuptools/blob/46fe69938e2b88fcad78d2dfbcfc701a4cfa6b3f/setuptools/command/sdist.py#L174 and https://github.com/pypa/setuptools/blob/46fe69938e2b88fcad78d2dfbcfc701a4cfa6b3f/setuptools/command/egg_info.py#L207